src/PrintProvider.jsx

```diff
@@ -43,28 +44,46 @@ export default class PrintProvider extends React.PureComponent {
     };
   }
 
-  regPrintable(key, node, isSingle) {
+  regHiddenAll() {
+    document.body.classList.add(s.hiddenAll);
+    this.hasSingle = true;
+  }
```

Comment:
the method has an incorrect naming since it `hasSingle` aware

Reply:
hm, I got 
anyways, let's consider `hideAll() {...}`

Reply:
What's wrong with naming ? 
This method responds to registration of class hidenAll in the body node.
hasSingle tells us than we have single ( main ) Print